docs/source/release_notes/lts/52_0.rst

```diff
@@ -163,3 +163,176 @@ When compared to the last LTS, we had:
+
+Usage
+^^^^^
+
+.. note:: As mentioned in previous section some previously core
+   features were turned into optional plugins. Do check your install
+   script if some command described here are missing on your system.
+
+Most workflows should work the same, although there are few little
+changes and a few obsoleted constructs which are still valid, but you
+should start using the new ones.
+
+The hard changes which does not provide backward compatibility:
+
+* Human result was tweaked a bit:
+    * The ``TESTS`` entry (displaying number of tests) was removed as
+      one can easily get this information from ``RESULTS``.
+    * Instead of tests time (sum of test times) you get job time
+      (duration of the job execution) in the human result
+* Json results also contain some changes:
+    * They are pretty-printed
+    * As ``cancel`` status was introduced, json result contain an entry
+      of number of canceled tests (``cancel``)
+    * url was renamed to id (url entry is to be removed in ``53.0`` so
+      this is actually a soft change with a backward compatibility support)
+* The ``avocado multilex|variants`` does not expect multiplex YAML
+  files as positional arguments, one has to use ``-m|--mux-yaml``
+  followed by one or more paths.
+* Test variants are not serialized numbers anymore in the default
+  ``yaml_to_mux`` (``multiplexer``), but ordered list of leaf-node names
+  of the variant followed by hash of the variant content
+  (paths+environment). Therefor instead of ``my_test:1`` you can get
+  something like ``my_test:arm64-virtio_scsi-RHEL7-4a3c``.
+* ``results.tap`` is now generated by default in job results along the
+  ``results.json`` and ``results.xml`` (unless disabled)
+* The ``avocado run --replay`` and ``avocado diff`` are unable to parse
+  results generated by ``36.4`` to this date. We should be able to
+  introduce such feature with not insignificant effort, but no one was
+  interested yet.
+
+And the still working but to be removed in ``53.0`` constructs:
+
+* The long version of the ``-m|--multiplex`` argument available in
+  ``avocado run|multiplex|variants`` was renamed to ``-m|--mux-yaml``
+  which corresponds better to the rest of ``--mux-*`` arguments.
+* The ``avocado multiplex`` was renamed to ``avocado variants``
+* The ``avocado multiplex|variants`` arguments were reworked to better
+  suite the possible multiple varianter plugins:
+
+    * Instead of picking between ``tree`` representation of list of
+      variants one can use ``--summary``, resp ``--variants`` followed
+      by verbosity, which supersedes ``-c|contents``, ``-t|--tree``,
+      ``-i|--inherit``
+    * Instead of ``--filter-only|--filter-out`` the
+      ``--mux-filter-only|--mux-filter-out`` are available
+    * The ``--mux-path`` is now also available in
+      ``avocado multiplex|variants``
+
+Test API
+^^^^^^^^
+
+Main features stayed the same, there are few new ones so do check our
```

Comment:
`a few` means few but significant amount, while `few` means insignificant amount. (at least that's what I had been told). Here I wanted to stress out it's really just minimal number of changed things.